static/bidder-info/amx.yaml

```diff
@@ -14,7 +14,12 @@ capabilities:
       - video
       - native
 userSync:
+  iframe:
+    url: "https://prebid.a-mo.net/isyn?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&s=pbs&cb={{.RedirectURL}}"
+    userMacro: "$UID"
```

Comment:
@nickjacob I am validating the user sync URL but looks like setuid endpoint is not being called. Trying the following URL - 
```
https://prebid.a-mo.net/isyn?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&s=pbs&cb=https%3A%2F%2Fib.adnxs.com%2Fsetuid%3Fbidder%3Damx%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%24UID
```

Reply:
Hi @gargcreation1992 sorry about that -- a change to support this wasn't rolled out fully on our server. It's fixed now / this should always fire the callback now.

Thanks!

Reply:
working now, thanks!